index.html

```diff
@@ -8,24 +8,29 @@
     <title>my-portfolio</title>
 </head>
 <body>
```

Comment:
[ OPTIONAL ] There is a horizontal scroll on the page for medium and small viewports which shows that some content is overflowing from the container. Please remove the horizontal scroll.

Reply:
Thank you so much. This is also fxed